hello_flask.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask, render_template
+from flask_sqlalchemy import SQLAlchemy
+
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://zainshah:123123@localhost/hello_flask'
+db = SQLAlchemy(app)
+
+
+@app.route('/')
+def display_posts():
+    from models import Post
```

Comment:
Imports in functions should not be used. Can use [Application Factories](http://flask.pocoo.org/docs/0.12/patterns/appfactories/) to avoid circular import.